api/Pipfile

```
@@ -50,7 +50,7 @@ wsgi-basic-auth = "*"
 python-decouple = "*"
 django-cron = "*"
 gunicorn = "*"
-sentry-sdk = "*"
+sentry-sdk = "==1.8.0"
```

After the Pipfile.lock is updated to the latest version, I think we can change this back to the `*` so that the lockfile is used to determine the version to install.

Pipenv is _very_ strange tool to me. It has conventions that I don't quite understand and limitations that seem odd to not be prioritized (like being able to updated individual tools). I wonder if we could switch to Poetry or PDM at some point... Both are also much faster than Pipenv.

```suggestion
sentry-sdk = "*"
```

---

[PDM](https://pdm.fming.dev/latest/#feature-highlights) looks quite promising, would like to try it soon to see if it's mature enough...